kuma/javascript/src/header/main-menu.jsx

```diff
@@ -218,11 +240,23 @@ const _MainMenu = ({ document, locale }: Props) => {
                                                 '{{PATH}}',
                                                 path
                                             )}
+                                            onClick={sendMenuItemInteraction}
+                                            onContextMenu={
+                                                sendMenuItemInteraction
+                                            }
                                         >
                                             {item.label} &#x1f310;
                                         </a>
                                     ) : (
-                                        <a href={item.url}>{item.label}</a>
+                                        <a
+                                            href={item.url}
+                                            onClick={sendMenuItemInteraction}
+                                            onContextMenu={
+                                                sendMenuItemInteraction
+                                            }
+                                        >
+                                            {item.label}
+                                        </a>
```

Comment:
This is what happens when I hover over the menu a bit:
<img width="1496" alt="Screen Shot 2019-12-23 at 2 52 54 PM" src="https://user-images.githubusercontent.com/26739/71377928-0e170a80-2594-11ea-8721-4cf7d13512d7.png">
(using Chrome and my debug build)

THere's something weird going on with that `onMouseOver` that's over-firing.